app/docker/views/networks/networksController.js

```diff
@@ -71,6 +72,12 @@ angular.module('portainer.docker').controller('NetworksController', [
           } else {
             $scope.networks = networks;
           }
+
+          $scope.networks = _.map($scope.networks, (network) => {
+            network.IPAM.IPV4Config = DockerNetworkHelper.getIPV4Configs(network.IPAM.Config);
+            network.IPAM.IPV6Config = DockerNetworkHelper.getIPV6Configs(network.IPAM.Config);
+            return network;
+          })
```

Comment:
Use `_.forEarch` instead of `_.map`

Use plural
Rename `network.IPAM.IPV4Config` to `network.IPAM.IPV4Configs`
Rename `network.IPAM.IPV6Config` to `network.IPAM.IPV6Configs`